tests/roles/glance_adoption/defaults/main.yaml

```diff
@@ -1,2 +1,2 @@
-# glance_backend can be 'local' or 'ceph'
-glance_backend: local
+# glance_backend can be 'local', 'ceph' or 'swift'
+glance_backend: swift
```

Comment:
@jistr so this should trigger `test_minimal` to deploy glance w/ a swift backend. While this would be the default behavior, I'm wondering if you think it's better to pass the var to the playbook itself.

Reply:
Yea the adoption has to follow the configuration of the environment we start with. I'm actually not sure what the Glance backend is when we deploy the Standalone without Ceph, but based on the defaults it's probably Swift, unless it's overriden somewhere else:

https://github.com/openstack/tripleo-heat-templates/blob/811bf370be3b9a4bb29b7234a6f6c16b8290146b/deployment/glance/glance-api-container-puppet.yaml#L227

So changing the test-minimal default to Swift backend makes sense to me. We could feed the variable to the playbook too, but i'm not sure if we want to test with `local` ever, now that we have Swift... So maybe it's ok just to change the default.

Reply:
+1 to keep `swift` as default (which is aligned w/ what TripleO provides). `local` shouldn't be tested at all, but we can keep that code because w/ a slightly change we can be ready to test the `NFS` backend.